src/SourceBuild/content/Directory.Build.props

```
@@ -178,6 +178,8 @@
     <SourceBuiltPrebuiltsTarballName>Private.SourceBuilt.Prebuilts</SourceBuiltPrebuiltsTarballName>
 
     <BaselineDataFile>$(ToolsDir)prebuilt-baseline.xml</BaselineDataFile>
+
+    <DotNetBuildArtifactsLogDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'log', '$(Configuration)', '$(MSBuildProjectName)'))</DotNetBuildArtifactsLogDir>
```

This should use ArtifactsLogDir: https://github.com/dotnet/arcade/blob/c8b7f328217240087a32e5dc9877b616ed96b7d8/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L47

```suggestion
    <DotNetBuildArtifactsLogDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsLogDir)', '$(MSBuildProjectName)'))</DotNetBuildArtifactsLogDir>
```

---

Updated.